addons/event_booth_sale/tests/test_event_booth_sale.py

```diff
@@ -72,3 +72,66 @@ def test_event_booth_prices_with_sale_order(self):
                          "Untaxed amount should be the sum of the booths prices ($200.0).")
         self.assertEqual(float_compare(sale_order.amount_total, 220.0, precision_rounding=0.1), 0,
                          "Total amount should be the sum of the booths prices with 10% taxes ($200.0 + $20.0).")
+
+        # Confirm the SO.
+        sale_order.action_confirm()
```

Comment:
Could be great to check what happens after confirm without invoicing ^^ (aka booth updates)